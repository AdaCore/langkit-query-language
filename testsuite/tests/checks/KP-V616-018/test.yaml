driver: 'checker'
rule_name: KP_V616_018
project: 'prj.gpr'
