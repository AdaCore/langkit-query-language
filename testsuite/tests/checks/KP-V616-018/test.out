p.ads:12:7: rule violation: possible occurrence of KP V616-018
12 |       F2 : Access_To_Array;  -- FLAG
   |       ^^

p.ads:13:7: rule violation: possible occurrence of KP V616-018
13 |       F3 : access String;    -- FLAG
   |       ^^

p.ads:25:10: rule violation: possible occurrence of KP V616-018
25 |          V : T_Vector_Access := null;   --  FLAG
   |          ^

